source/data_at_rest_encryption.rst

```diff
@@ -77,10 +54,10 @@ the encryption. By default, the |opt.encryption-cipher-mode| option uses the
    --outputPath              The path to save the decrypted rollback file
    ========================  ==================================================================================
 
-.. rubric:: Important Configuration Options
+.. rubric:: Important Configuration Options When Using Key File Encryption
```

Comment:
`--enableEncryption` and `--encryptionCipherMode` are used for both keyfile and vault, so they are common options. Others depends where you store the key.